tethys_portal/_version.py

```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
```

Comment:
I get the feeling that we should not track this in version control

Reply:
Removed